client/asset/btc/btc.go

```diff
@@ -911,7 +933,7 @@ func (btc *ExchangeWallet) fundedTx(coins asset.Coins) (*wire.MsgTx, uint64, []o
 // are NOT manually unlocked because they're auto-unlocked when the transaction
 // is broadcasted.
 func (btc *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin, uint64, error) {
-	contracts := make([][]byte, 0, len(swaps.Contracts))
+	contractScripts := make([][]byte, 0, len(swaps.Contracts))
```

Comment:
Made that change because the loop that appends to this slice below uses the variable name `contract` for each `asset.Contract` to be swapped and uses `pkScript` within the loop for the contract script which could be misleading. So I renamed `pkScript` to `contractScript` in the loop and `contracts` to `contractScript` to disambiguate `contract` as asset.Contract from the []byte script and `swap.Contracts` as []asset.Contracts from the [][]byte collection.

To confirm, the contract script isn't the `pkScript` or is it? Elsewhere it is treated as the redeem script (in AuditConract:
https://github.com/decred/dcrdex/blob/aa65868f238a08395947ec6590455104f8636dcc/client/asset/btc/btc.go#L1187-L1192.

Reply:
The contract is the redeem script for a p2sh output, which has as it's pkScript a different script of the format OP_HASH160 <contract script hash> OP_EQUAL.  So calling the contract a redeem script consistently is good.

Reply:
I'm not sure it is _explicitly_ called a redeemScript anywhere, only implicitly as in the above linked example. I've replaced `pkScript` in the loop with `contractScript`, prefer `redeemScript` instead?